### ISSUE TITLE
warm step

### ISSUE THREAD

**Issue by ShaniGam** [twangnh/Distilling-Object-Detectors#13]
Can you please explain the intuition for using `warm_step=200` for only 1 epoch? It doesn't seem like enough for meaningful training without distillation. What happens if I use the distillation loss from scratch?

**Comment by twangnh** [comment]
can you rephrase your question?

**Comment by ShaniGam** [comment]
The warm step is not mentioned in the paper. Does it improve the result?

**Comment by twangnh** [comment]
no, warm up is not related to distillation, it is used for stable training
